docker-compose.yml

```
@@ -8,30 +8,25 @@ secrets:
     file: ./src/secrets/quote.txt
 
 services:
-  example:
+  pca-report-library:
     # Run the container normally
     build:
       # VERSION must be specified on the command line:
       # e.g., --build-arg VERSION=0.0.1
       context: .
       dockerfile: Dockerfile
-    image: cisagov/example
+    image: cisagov/pca-report-library
     init: true
     restart: "no"
     environment:
       - ECHO_MESSAGE=Hello World from docker-compose!
-    ports:
-      - target: 8080
-        published: 8080
-        protocol: tcp
-        mode: host
     secrets:
       - source: quote_txt
         target: quote.txt
 
-  example-version:
+  pca-report-library-version:
     # Run the container to collect version information
-    image: cisagov/example
+    image: cisagov/pca-report-library
```

We encourage using version tags on all of our Docker images whenever possible:
```suggestion
    image: cisagov/pca-report-library:0.0.1
```